docs/source/conf.py

```diff
@@ -1,38 +1,45 @@
-#!/usr/bin/env python
```

Comment:
This has been built automatically by sphinx-quickstart, but I believe all the params are correct there.